src/log.rs

```diff
@@ -4,6 +4,10 @@
 
 use crate::context::Context;
 
+mod logging_stream;
```

Comment:
I'd rename it to just `stream`. Though doesn't matter much because `LoggingStream` is re-exported